src/shared/Atlassian.Bitbucket/BitbucketHostProvider.cs

```diff
@@ -145,15 +178,14 @@ public async Task<ICredential> GetCredentialAsync(InputArguments input)
             }
             else
             {
-                // TODO: should we try and compute if the AT has expired and use it?
-                // This needs support from the credential store to record the expiry time!
+                _context.Trace.WriteLine($"    Found refresh token: {refreshToken} ");
 
                 // It's very likely that any access token expired between the last time we used/stored it.
                 // To ensure the AT is as 'fresh' as it can be, always first try to use the refresh token
                 // (which lives longer) to create a new AT (and possibly also a new RT).
```

Comment:
I don't think this comment is correct now, since if we're here then we've already found no such AT, or found an AT that doesn't work.

```suggestion
```